src/implementations/vanilla/components/breadcrumb/breadcrumb.js

```diff
@@ -208,6 +208,7 @@ export class Breadcrumb {
    */
   handleFullExpand() {
     this.element.classList.remove('ecl-breadcrumb--collapsed');
+    this.element.classList.add('ecl-breadcrumb--wrap');
```

Comment:
We should remove this class we add on destroy() if it's present, it's not really going to happen but imagine the case where the instance is destroyed and re-initialized, it would not prevent wrapping.